packages/paste-website/src/pages/introduction/about-paste/index.mdx

```diff
@@ -14,7 +14,7 @@ import {InlineCode} from '../../../components/Typography';
 
 export const pageQuery = graphql`
   {
-    mdx(fields: {slug: {eq: "/introduction/about-paste/"}}) {
+    mdx(frontmatter: {slug: {eq: "/introduction/about-paste"}}) {
```

Comment:
question: are slugs not supposed to end w/ a `/`? A lot of them do, so I'm wondering why this one is taken out/ if that was intentional.

Reply:
It's not a case of whether it should or shouldn't, it doesn't really matter.

What matters is that the slug you provide in the Frontmatter at the top of the file, _exactly_ matches the slug you provide in the graphQL query. This one didn't.

The old version of Gatsby kinda just worked still, but this new one threw an error as it's being less forgiving of our silly little mistakes we make.